### DISCUSSION TITLE
Different results to example notebooks

### DISCUSSION THREAD

**Discussion by galacticStorm** [NanoComp/meep#2418] [Q&A] [answered]
Hi! This is my first writing a post like this, so please be patient. In short I installed meep on my ubuntu 22.04 WSL terminal using conda exactly as instructed in the documentations. I then ran some of the example notebooks and started to notice strange results. I thought it was due to the subnormals being changed to zero due to some WSL shenanigans, and after a lengthy tirade I realized that meep itself changes subnormals to zeros (the [example notebooks](https://nbviewer.org/github/NanoComp/meep/blob/master/python/examples/3rd-harm-1d.ipynb) clearly show this as well). By now you may be wondering what my strange results are like, focusing on the "Third Harmonic Generation" notebook (linked before) my harmonics are off after the first 8 significant figures and the my graph (top) looks quite different to the expected one (below) https://imgur.com/a/G1irFdm. Note my conda environment has nothing other than what was said in the documentation + jupyter (also from conda-forge). Any ideas what went wrong? Thanks in advance!!

**Comment by oskooi** [comment, marked as answer]
This seems to be related to #2408 which was fixed by #2413. The tutorial notebook was updated in #2411. However, the Conda package that you are using ([version 1.25](https://github.com/NanoComp/meep/releases)) has *not* yet been updated with these changes. This is why you are observing similar results reported in #2408.

The Conda package will be updated with the release of version 1.26 due out soon.
